opsgenie/resource_opsgenie_incident_template_test.go

```diff
@@ -0,0 +1,167 @@
+package opsgenie
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	ogClient "github.com/opsgenie/opsgenie-go-sdk-v2/client"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/incident"
+)
+
+var randomName = acctest.RandString(6)
```

Comment:
we should get this from config value no need to use global variable for this

Reply:
Changes done. removed randomName variable.